### PR TITLE
Add apt-transport-s3 to ubuntu-ruby base images

### PR DESCRIPTION
**Pull request by joecorcoran**
So that any containers based on these images can use our s3 repositories
to install software that is not in the regular deb repositories.

In the first case I will use it to add ffmpeg to content-ingestion.

### CODE REVIEW

File: ubuntu-14.04-ruby-1.9.3-p484/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,9 @@
 FROM ubuntu:14.04
 
+COPY apt-transport-s3.deb /tmp/
+RUN dpkg -i /tmp/apt-transport-s3.deb
+RUN apt-get install -f
```
Comment (MrPink) [inline ubuntu-14.04-ruby-1.9.3-p484/Dockerfile]: why is this required?

File: ubuntu-14.04-ruby-1.9.3-p484/Dockerfile [hunk 2]
```diff
@@ -1,5 +1,9 @@
 FROM ubuntu:14.04
 
+COPY apt-transport-s3.deb /tmp/
```
Comment (fidothe) [inline ubuntu-14.04-ruby-1.9.3-p484/Dockerfile]: What are we using with Docker and ruby 1.9?

File: ubuntu-14.04-ruby-1.9.3-p484/Dockerfile (hunk 1 above)
Comment (chdorner) [inline ubuntu-14.04-ruby-1.9.3-p484/Dockerfile]: Apparently `dpkg -i` doesn't run after install hooks and stuff like that. Just found that out on stack overflow somewhere.

File: ubuntu-14.04-ruby-1.9.3-p484/Dockerfile (hunk 2 above)
Comment (chdorner) [inline ubuntu-14.04-ruby-1.9.3-p484/Dockerfile]: `express-api` used to use 1.9 when I did the Docker stuff for it, then @MrPink switched it to 2.0. We should keep it here though, when we do Docker for metrics or one of those older projects.

**Pull request merged**